pom.xml

```diff
@@ -282,9 +282,9 @@
           </jvmArgs>
           <javacArgs>
             <javacArg>-source</javacArg>
-            <javacArg>${java.version}</javacArg>
+            <javacArg>1.7</javacArg>
```

Comment:
+1, needs to be changed. Although, is there a reason to move away from the `${java.version}` property here, if we're generally moving back towards using the parameters throughout the POM? I would prefer going back to `${java.version}` on this line.